### PR TITLE
Ticket Price Calculator modal:  replaced disabled base price with a label.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
Fixes this issue: https://github.com/eventespresso/event-espresso-core/issues/1743

## How has this been tested
Locally:
image

## Checklist

* [ ] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [ ] User input is adequately validated and sanitized
* [ ] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [ ] My code is tested.
* [ ] My code follows the Event Espresso code style.
* [ ] My code has proper inline documentation.
* [ ] My code accounts for when the site is in Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: assets/src/components/form/layouts/two-column-admin/two-column-admin.css [hunk 1]
```diff
@@ -483,6 +483,10 @@
     margin-top: 3px;
 }
 
+.ee-two-column-admin .ee-visible-label {
+    padding-left: .4rem;
```
Comment (nicolad) [inline assets/src/components/form/layouts/two-column-admin/two-column-admin.css]: @tn3rb I am not :100: % happy with this approach.
If there is a better mechanism of handling spacing in EE, please do suggest.
On the previous project, we had something similar to https://getbootstrap.com/docs/4.3/utilities/spacing/

**Comment by tn3rb** [comment]
@nicolad 
so originally the price calculator showed the "base price" label for the first price option, but that broke at some point. 
I just discovered that the actual problem is that the price objects no longer have a `price.prtId` property because of some changes that happened in the models a little while ago, and that property is now named `price.PRT_ID`.

So can I get you to revert your changes you've made so far (sorry) and instead run a find and replace on the `/assets/src/` folder and change usages of `prtId` to `PRT_ID`? Another option if you prefer, is to simply invalidate this PR, delete the branch, and then create a new one.

Your choice, and my apologies for making you redo this.

**Comment by tn3rb** [comment]
oh and here's a screenie of what the price calculator looks like when the base price is detected properly:

price calc base price

**Comment by nicolad** [comment]
> @nicolad
> so originally the price calculator showed the "base price" label for the first price option, but that broke at some point.
> I just discovered that the actual problem is that the price objects no longer have a `price.prtId` property because of some changes that happened in the models a little while ago, and that property is now named `price.PRT_ID`.
> 
> So can I get you to revert your changes you've made so far (sorry) and instead run a find and replace on the `/assets/src/` folder and change usages of `prtId` to `PRT_ID`? Another option if you prefer, is to simply invalidate this PR, delete the branch, and then create a new one.
> 
> Your choice, and my apologies for making you redo this.

I reverted commit and pushed changes here to have more context related to the fix.

**Pull request merged**